app/Models/Litter/Categories/Dogshit.php

```diff
@@ -13,20 +13,11 @@ class Dogshit extends LitterCategory
 
     protected $table = 'dogshit';
 
-    protected $fillable = [
-        'poo',
-        'poo_in_bag'
-    ];
-
-    /**
-     * Pre-defined litter types available on this class
-     */
-    public function types ()
+    public function types(): array
```

Comment:
It would help to have these as static methods. We can then use $types = Photo::types(); anywhere to get the types

Reply:
We can't do that right away. In order to get the types for a category, we need the category class name. But on a photo model, we only have the categories() method as a static method. The only current way to get to the class name of the categories is by instantiating them, like `$photo->$category->make();`. Otherwise, we'll have to construct a method that returns class names, like this:
```php
 public static function getCategoryClassName($category): string
    {
        $classNames = [
            'smoking' => Smoking::class,
            'art' => Art::class,
            ...
        ];
        return $classNames[$category];
    }
```
So that we can call `::types()` on it. But I don't like this approach at all. It would make the Photo model more complex than it is, we'd be duplicating the categories. I converted all the types() methods on the categories into static methods, but if we try to use them statically it will be a step back in refactoring. I propose we keep them as static methods but don't change the other parts of the code because of that. What do you think?